specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -774,6 +1331,66 @@
         }
       }
     },
+    "AcsRouterJobEventBaseProperties": {
+      "description": "Schema of common properties of all Router Job events",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "queueId": {
+          "description": "Router Job events Queue Id",
+          "type": "string"
+        },
+        "labels": {
+          "description": "Router Job events Labels",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "tags": {
+          "description": "Router Jobs events Tags",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "AcsRouterWorkerEventBaseProperties": {
```

Comment:
This should be named AcsRouterWorkerEventData

Reply:
same here